Essentials/src/config.yml

```diff
@@ -713,22 +713,28 @@ protect:
     water-bucket-flow: false
     fire-spread: true
     lava-fire-spread: true
+    lava-itemdamage: true
```

Comment:
I would recommend this be set to `false` by default, because this is a change which is going to take many server owners by surprise, especially if they are trying to run a fairly standard vanilla survival server.